docs/src/pages/ui/primitives/select/index.page.mdx

```diff
@@ -110,7 +110,7 @@ import '@aws-amplify/ui-react/styles.css';
 
 ### Variation
 
-Select comes in 3 variants, `filled`, `flushed` and none(default). Pass the `variation` prop and set it to either of these values.
+Select comes in 2 variants, `flushed` and none(default). Pass the `variation` prop and set it to either of these values.
```

Comment:
I think we decided in the meeting recently to change `flushed` to `quiet` to be more semantic.